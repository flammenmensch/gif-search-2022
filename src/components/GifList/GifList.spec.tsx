import { render, screen } from '@testing-library/react';
import { GifList } from './index';

// Jest
test('It renders', () => {
  render(<GifList items={[]} />);

  const list = screen.getByRole('list');
  expect(list).toBeInTheDocument();
});

test('It matches snapshot', () => {
  const result = render(<GifList items={[]} />);
  expect(result.asFragment()).toMatchSnapshot();
});
