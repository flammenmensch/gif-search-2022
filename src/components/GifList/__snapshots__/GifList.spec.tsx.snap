// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It matches snapshot 1`] = `
<DocumentFragment>
  <ul
    class="gif-list"
    role="list"
  />
</DocumentFragment>
`;
